projects/vdk-plugins/vdk-meta-jobs/src/vdk/plugin/meta_jobs/cached_data_job_executor.py

```diff
@@ -0,0 +1,130 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import json
+import logging
+import os
+import time
+from typing import Dict
+
+from taurus_datajob_api import ApiException
```

Comment:
taurus_datajob_api? what is Taurus?

Reply:
Taurus is the second astrological sign in the modern zodiac. 

It appears also that the python API client library generated by the control service uses this as a package name - https://github.com/vmware/versatile-data-kit/blob/main/projects/control-service/projects/model/apidefs/datajob-api/config-python.json#L2